Shaders/MeshChain.cginc

```diff
@@ -1,5 +1,6 @@
 #ifndef MESH_CHAIN
 #define MESH_CHAIN
+// UNITY_SHADER_NO_UPGRADE
```

Comment:
Gotcha. Seems weird to me that a comment would actually change how Unity updates scripts, but then again I haven't done any shader programming in Unity.